libraries/ui/src/default-config/tailwind.ts

```diff
@@ -73,17 +73,20 @@ export const withDefaultBlueDotTailwindConfig = (config: Partial<Config> & { con
         },
         fontSize: {
           /* Design System Font Sizes */
-          'size-s': ['0.75rem' /* 12px */, { lineHeight: '1rem' /* 16px */ }], // equivalent to xs
-          'size-m': ['0.875rem' /* 14px */, { lineHeight: '1.25rem' /* 20px */ }], // equivalent to sm
-          'size-l': ['1.5rem' /* 24px */, { lineHeight: '2rem' /* 32px */ }], // equivalent to 2xl
-          'size-xl': ['3rem' /* 48px */, { lineHeight: '1' }], // equivalent to 5xl
+          'size-xxs': 'var(--text-size-xxs)',
+          'size-xs': 'var(--text-size-xs)',
+          'size-sm': 'var(--text-size-sm)',
+          'size-md': 'var(--text-size-md)',
+          'size-lg': 'var(--text-size-lg)',
+          'size-xl': 'var(--text-size-xl)',
           ...config.theme?.extend?.fontSize,
         },
         spacing: {
-          'max-width': '1440px',
-          'min-width': '388px',
-          gutter: '16px',
-          'gutter-sm': '8px',
+          'max-width': 'var(--max-width)',
+          'spacing-x': 'var(--spacing-x)',
+          'spacing-y': 'var(--spacing-y)',
+          column: 'var(--column)',
```

Comment:
`column` size is still a question mark for me. It does not seem like the designs currently follow a spec with `column width 210px` (and `6` columns per row). If we do not currently need this, I suggest we leave it out so it does not cause confusion.